frontend/public/components/monitoring/targets.tsx

```diff
@@ -293,9 +320,34 @@ const REDUX_ID = 'monitoringTargets';
 
 const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.lastError });
 
+const getWatchError = (monitorsWatchContext, t) => {
+  const loadErrorIndex = 2;
+  const watchError = monitorsWatchContext[loadErrorIndex];
+  const authInfo =
+    'https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html';
+
+  if (watchError) {
+    if (watchError.code === 403) {
+      return (
+        <>
+          <p> {watchError.message} </p>
+          <ExternalLink href={authInfo} text={t('Learn More')} />
+        </>
+      );
+    }
+    return `${watchError.message}`;
+  }
+};
+
 const List: React.FC<ListProps> = ({ loaded, loadError, targets }) => {
   const { t } = useTranslation();
 
+  const [showServiceMonitorError, setShowServiceMonitorError] = React.useState(true);
+  const [showPodMonitorError, setShowPodMonitorError] = React.useState(true);
```

Comment:
We have a `useBoolean` hook that I find useful for this situation.

```suggestion
  const [showServiceMonitorError, , , hideServiceMonitorError] = React.useBoolean(true);
  const [showPodMonitorError, , , hidePodMonitorError] = React.useBoolean(true);
```

One advantage is that the `hideServiceMonitorError` and `hidePodMonitorError` functions will now be the same function each time this component is rendered.
(Currently `AlertActionCloseButton`'s `onClose` props are functions that are recreated each time. In this case it probably doesn't matter, but sometimes that can significantly slow render time by causing components to render again unnecessarily.)